tractor/_runtime.py

```diff
@@ -319,7 +319,7 @@ async def _invoke(
             BrokenPipeError,
         ):
             # if we can't propagate the error that's a big boo boo
-            log.error(
+            log.exception(
```

Comment:
This is important to show *what* traceback wasn't able to be propagated to the far end actor's caller task.